components/Radio/src/stories/bem.stories.mdx

```diff
@@ -74,9 +80,48 @@ import readme from "../../README.md";
       <label class="denhaag-form-control-label denhaag-form-control-label--disabled">
```

Comment:
{() => ( 
Volgens mij mag de react uit de bem, hiervoor is een aparte react.stories.mdx file